I've completed Homework-19

AParkizin opened the pull request:
Test was passed successfully

parkizin committed:
I've completed Homework-19

Test was passed successfully

Berichicko commented on src/test/java/Homework19.java:
```diff
@@ -0,0 +1,39 @@
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class Homework19 extends BaseTest {
+    @Test
+    public void deletePlaylist() throws InterruptedException{
+        String expectedDeletedPlaylistMsg = "Deleted playlist \"Test Pro Playlist.\"";
```
How can I beautify code in IntelliJ:

The formatting shortcuts in Intellij IDEA are :
For Windows : Ctrl + Alt + L.
For Ubuntu : Ctrl + Alt + Windows + L.
For Mac : ⌥ (Option) + ⌘ (Command) + L.

![image](https://user-images.githubusercontent.com/104761482/209774134-9e63bfed-d768-4455-a860-028305c85e17.png)

Berichicko commented on src/test/java/Homework19.java:
```diff
@@ -0,0 +1,39 @@
+    public void deletePlaylist() throws InterruptedException{
+        String expectedDeletedPlaylistMsg = "Deleted playlist \"Test Pro Playlist.\"";
+
+    provideEmail("demo@class.com");
+    providePassword("te$t$tudent");
```
You got this problem, please correct it.

![image](https://github.com/AParkizin/A57-QA-Automation-Framework/assets/104761482/1b54992e-11e3-487e-80f2-a94e80b09a05)

HINT

Different data for login and password

For example 

BaseTest

![image](https://github.com/AParkizin/A57-QA-Automation-Framework/assets/104761482/410bccdb-3e8d-4510-bbb9-45d8c7643b7f)

Homework19

![image](https://github.com/AParkizin/A57-QA-Automation-Framework/assets/104761482/7cf037b7-8224-4a93-abe2-db0f1175cf95)

SOLUTION

![image](https://github.com/AParkizin/A57-QA-Automation-Framework/assets/104761482/cdff90d3-2cc4-4b74-a818-c6b0e13c58d0)

![image](https://github.com/AParkizin/A57-QA-Automation-Framework/assets/104761482/0d97e753-e201-49bd-847f-cb6272ba4557)

parkizin committed:
Homework-19 was fixed

I change credentials: In "BaseTest.java" I added "password" and "email" instead of my credentials. In "Homework19.java" I deleted wrong credentials and added mine.
